pkg/admin/prerun/health.go

```diff
@@ -26,42 +21,31 @@ type HealthInfo struct {
 	BootID       string `json:"boot_id"`
 }
 
-func (hi *HealthInfo) BackupName() data.BackupName {
-	name := fmt.Sprintf("%s_%s", hi.DeploymentID, hi.BootID)
-
-	if hi.IsHealthy() {
-		return data.BackupName(name)
-	}
-
-	return data.BackupName(fmt.Sprintf("%s_unhealthy", name))
-}
-
-func (hi *HealthInfo) IsHealthy() bool {
-	return hi.Health == "healthy"
-}
-
-func getHealthInfo() (*HealthInfo, error) {
-	if exists, err := util.PathExistsAndIsNotEmpty(healthFilepath); err != nil {
-		return nil, err
-	} else if !exists {
-		return nil, errHealthFileDoesNotExist
+// updateHealthInfo updates health.json files with hardcoded "healthy" and
+// deployment and boot IDs from provided argument.
+func updateHealthInfo(vf versionFile) error {
+	// health.json in kept in place to support rollback to 4.14.0~rc.1 and earlier,
+	// as lack of the file would result in MicroShift neither backing up the data
+	// nor restoring a backup and we want to always create a backup.
+	// We should be able to remove the health.json file in 4.16 or even 4.15
+	// if we state that upgrading from rc.0 and rc.1 directly to 4.15 is not supported,
+	// but should be done with intermediate step to released 4.14.0.
```

Comment:
We should not plan on being able to have a restriction like that, unfortunately.